src/vs/workbench/browser/layout.ts

```diff
@@ -167,7 +167,11 @@ export abstract class Layout extends Disposable implements IWorkbenchLayoutServi
 		}
 		if (this.isVisible(Parts.TITLEBAR_PART)) {
 			top += this.getPart(Parts.TITLEBAR_PART).maximumHeight;
-			quickPickTop = this.titleService.isCommandCenterVisible ? quickPickTop : top;
+			quickPickTop = top;
+		}
+		// If the command center is visible then the quickinput should go over the title bar and the banner
+		if (this.titleService.isCommandCenterVisible) {
+			quickPickTop = 0;
```

Comment:
Unrelated change? Will also overlay the banner which I believe is unwanted?

Reply:
@jrieken I actually wanted that. because otherwise the command center disappears and it looks weird.

Reply:
makes sense to my but maybe /cc @daviddossett given emotion with the banner were high. I agree for better looks but I believe the point of the banner is to not be permanent?

Reply:
Not sure I follow—what is the effect of this?

Reply:
@daviddossett so you know we have this banner:
<img width="656" alt="image" src="https://user-images.githubusercontent.com/2644648/198129934-fa0e66bc-4478-45b1-9beb-04fa3b8b906e.png">

today, if you have that banner (be in an untrusted workspace) and you launch the command center this happens:
<img width="799" alt="image" src="https://user-images.githubusercontent.com/2644648/198130187-9117ed24-676e-455a-a699-609e19fe6d7b.png">
Actually I think that's a bug because really what the code was intending to do, originally, was have the quick pick be _under_ the banner so it didn't cover it basically with the idea that "overlaying anything over the banner is bad"... 

My change says "covering the banner for the quick pick is ok" and has the quick pick do what it does today in an untrusted workspace.

Reply:
I think covering it is totally ok personally. It would be strange to decouple the command center button/input from the quick pick itself.

I don't actually see the titlebar element in the second screenshot. Is that intentional?

Reply:
Yeah it is hidden when you click on it since it is often longer than the quickpick itself.

Reply:
I think we're all in agreement. I'll defer to @jrieken for a re-review